core/dictgen/src/rootcling_impl.cxx

```diff
@@ -3526,8 +3526,9 @@ gOptModuleByproducts("mByproduct", llvm::cl::ZeroOrMore,
                      llvm::cl::Hidden,
                      llvm::cl::desc("The list of the expected implicit modules build as part of building the current module."),
                      llvm::cl::cat(gRootclingOptions));
+// Really llvm::cl::Required, will be changed in RootClingMain below.
```

Comment:
I am confused, could you elaborate more why we cannot make this globally? Eg, why it worked before and does not work now.

Reply:
It worked because `rootcling` was the only one parsing options using LLVM's global option registration. To enable `-mllvm`, we must avoid that there are required positional arguments. In general, it would be better if `rootcling` didn't use LLVM's option parsing infrastructure at all, or that the options are only registered in the binaries. The latter is not possible due to how we confine symbols in `libCling`.